<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>balance_dia</class>
 <widget class="QWidget" name="balance_dia">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>724</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Balance del día</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>img/logo2.png</normaloff>img/logo2.png</iconset>
  </property>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>320</y>
     <width>51</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>img/logo2.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>211</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Fecha: 21/07/2021 13:25:48</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>10</y>
     <width>88</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Imprimir</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>171</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Registrados en el día:</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>611</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
